scripts/patches/README.md

```diff
@@ -34,8 +34,9 @@ Example scripts
 
     SQ_HOME=$1
 
-    echo "enabling debug in conf/sonar.properties, listening on port 5005"
-    set_property sonar.web.javaAdditionalOpts -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005 "$SQ_HOME/conf/sonar.properties"
+    port=5005
+    echo "enabling debug in conf/sonar.properties, listening on port $port"
+    set_property sonar.web.javaAdditionalOpts -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=$port "$SQ_HOME/conf/sonar.properties"
```

Comment:
no quotes around `sonar.web.javaAdditionalOpts` and `-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=$port` ?

I start to do it because vi colors argument when quotes are present and doesn't when they are missing.
Probably, it's would also be relevant in this case because of `$port`

Reply:
A space in `$port` is a highly unlikely mistake. But since the parameter is quite long, it's not obvious at a glance that it's safe to not quote, so I added the quotes now.